osbenchmark/worker_coordinator/runner.py

```diff
@@ -1080,23 +1084,33 @@ def _get_field_value(content, field_name):
                     return _get_field_value(content["_source"], field_name)
                 return None
 
-            def calculate_recall(predictions, neighbors, top_k):
```

Comment:
we can't remove top_k. It was added to support top k where k < len(neighbors[0])

Reply:
Created another individual function for calculating radial search recall